docs/deployment/cumulus-distribution.md

```diff
@@ -149,10 +153,18 @@ Waiting for connections...
 
 In another terminal window, open a tunnel with port forwarding using your chosen port from above (e.g., 9000):
 
+##### Using SSH keypair authentication
+
 ```bash
 ssh -4 -p 6000 -N -L <port>:<api-gateway-host>:443 ec2-user@127.0.0.1
 ```
 
+##### Using PKCS11Provider
```

Comment:
Do we need a note about what this is? Something tying it directly to PIV access?

Reply:
What about just a note that PKCS11 === PIV? 

e.g. `Using PKCS11Provider (for users requiring PIV authentication)`

Reply:
I'm being annoying because I'm the target audience for this heading and I didn't know what a PKCS11Provider is 😄